src/main/java/frc/robot/constants/Constants.java

```diff
@@ -22,7 +23,9 @@ public class Constants {
     public static final String CANIVORE_CAN = "CANivore";
     public static final String RIO_CAN = "rio";
 
-    public static final boolean DO_LOGGING = false;
+    // Logging 
+    public static final boolean DO_LOGGING = true;
+    public static final LogLevel LOG_LEVEL = LogLevel.DEBUG;
```

Comment:
You could keep the current code, although it might be better to have a LogLevel.NONE instead of a separate boolean to disable it.